CHANGES.md

```diff
@@ -1,5 +1,45 @@
 # Changes
 
+## stable-2.12.2
+
+This stable release fixes an issue with CNI chaining that was preventing the
+Linkerd CNI plugin from working with other CNI plugins such as Cilium. It also
+fixes some sections of the Viz dashboard appearing blank, and adds an optional
+PodMonitor resource to the Helm chart to enable easier integration with the
+Prometheus Operator. Several other fixes are included.
+
+* Proxy
+  * Fixed proxies emitting some duplicate inbound metrics
```

Comment:
should this mention which metrics could be duplicated?

Reply:
I'm not familiar with this fix, perhaps you could expand on that?